numba/np/ufunc/gufunc.py

```diff
@@ -133,8 +136,7 @@ def __call__(self, *args):
         # at this point we know the gufunc is a dynamic one
         ewise = self._get_ewise_dtypes(args)
         if not (self.ufunc and ufunc_find_matching_loop(self.ufunc, ewise)):
-            self._is_dynamic = True
```

Comment:
This is odd. Is this fixing an oversight unrelated to kwarg processing?

Reply:
Correct